librad/src/git/include.rs

```diff
@@ -101,11 +110,22 @@ impl<Path> Include<Path> {
         {
             let mut config = git2::Config::open(tmp.path())?;
             for remote in &self.remotes {
-                let (key, url) = url_entry(&remote);
-                config.set_str(&key, &url.to_string())?;
-
-                let (key, fetch) = fetch_entry(&remote)?;
-                config.set_str(&key, &fetch)?;
+                let (url_key, url) = url_entry(&remote);
+
+                let (fetch_key, fetch) = match fetch_entry(&remote) {
```

Comment:
I'm having second thoughts here: why are we ignoring misconfigured remotes instead of returning an error? @FintanH

Reply:
Theoretically, we can only add correctly configured remotes using https://github.com/radicle-dev/radicle-link/pull/412/files#diff-b2cdbd6dedfb9c88ab61da2cb96e6adc3c867e1b9662fb3f94d34436bb484f54R168. But this is really a poor excuse.

Should we `expect` or return an error instead?